docs/simplesamlphp-upgrade-notes-2.0.md

```diff
@@ -144,3 +145,53 @@ $x = \SimpleSAML\Utils\Arrays::arrayize($someVar)
 
 [1]: https://github.com/simplesamlphp/simplesamlphp/wiki/Migrating-translations-(pre-migration)
 [2]: https://github.com/simplesamlphp/simplesamlphp/wiki/Twig:-Migrating-templates
+
+## Upgrading and EntityIDs
+
+If you still have your 1.x installation available, the entityID you
+are using for your SP and IdP should be available in
+module.php/core/frontpage_federation.php location on your
+SimpleSAMLphp server.
+
+An EntityID is a unique identifier for your SP or IdP. It has to be
+unique, like an IP-address has to be unique in your network. You might
+like to use your host name and some additional path information for
+the EntityID.
```

Comment:
We have some instructions in the docs on how to pick one. Maybe keep this section focussed on the changes between 1.19 and 2.x, as the document is about upgrading. People should not land here that will need to pick a new entityID from scratch - they are helped in the other parts of the docs (and if they are not we should fix that there).

Reply:
ok removed this paragraph. It is a bad idea if this happens to be favoured by the search engines over the existing part of the docs discussing EntityIDs.